src/PlacesAutocomplete.js

```diff
@@ -236,7 +236,12 @@ class PlacesAutocomplete extends Component {
         id="PlacesAutocomplete__root"
         style={this.inlineStyleFor('root')}
         className={this.classNameFor('root')}>
-        <input {...inputProps} />
+        <input {...inputProps} aria-describedby="PlacesAutocomplete__autocomplete-description"/>
+        <span 
+          id="PlacesAutocomplete__autocomplete-description"
+          style={this.inlineStyleFor('autocompleteDescription')}
```

Comment:
@kenny-hibino I have been trying to think of any scenarios in which the user may want to see that span... but I haven't come up with any use cases either. I will submit another PR